sphinx/domains/changeset.py

```diff
@@ -84,10 +86,16 @@ def run(self) -> List[Node]:
 
             para = cast(nodes.paragraph, node[0])
             para.insert(0, nodes.inline('', '%s: ' % text, classes=classes))
+        elif len(node) > 0:
+            # the content does not starts with a paragraph
```

Comment:
s/starts/start/

And I am hesitating with s/content does/contents do/ but not a native speaker :)

Reply:
Thanks! I'll follow your intuition :-)